src/widgets/_views.scss

```diff
@@ -17,4 +17,18 @@ scrolledwindow {
 .view {
     background-color: bg-color(1);
     color: $fg-color;
+
+    &.terminal {
+        background-color: #252e32;
+        color: #94a3a5;
+
+        & text {
+            font-family: monospace;
```

Comment:
Looks like this actually needs to apply to `.terminal` , not `text` to take effect

Reply:
Fixed.